recipes/dgl/meta.yaml

```
@@ -0,0 +1,88 @@
+{% set name = "dgl" %}
+{% set version = "1.1.0" %}
+{% set torch_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/dmlc/dgl
+  git_rev: {{ version }}
+  patches:
+    - conda-build.patch
+
+build:
+  skip: true  # [(not linux) or (cuda_compiler_version in (undefined, 'None'))]
+  number: 0
+  rpaths:
+    - lib/
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux64]
+    - "$RPATH/libtorch_cpu.so"
+    - "$RPATH/libc10.so"
+    - "$RPATH/libtorch.so"
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version not in (undefined, 'None')]
+    - sysroot_linux-64  2.17  # [linux64]    
+    - cmake
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - dlpack ==0.7
+    - dmlc ==0.5
+    - gklib ==5.1.1
+    - metis ==5.1.0
+    - nanoflann
+    - parallel-hashmap
+    - pytorch * *{{ torch_proc_type }}*
```

These should all be in `host`

```suggestion
    - dlpack ==0.7
    - dmlc ==0.5
    - gklib ==5.1.1
    - metis ==5.1.0
    - nanoflann
    - parallel-hashmap
    - pytorch * *{{ torch_proc_type }}*
```